src/Storage/Device/OracleObject.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Utopia\Storage\Device;
+
+use Exception;
+use Utopia\Storage\Device;
+use Utopia\Storage\Storage;
+
+class OracleObject extends Device
+{
+    /**
+     * Oracle Object Storage configuration options
+     */
+    private array $config;
+
+    /**
+     * OracleObject constructor.
+     *
+     * @param string $root
+     * @param array $config Configuration options for Oracle Object Storage
+     */
+    public function __construct(string $root = '', array $config = [])
+    {
+        $this->root = $root;
+        $this->config = $config;
```

Comment:
Let's not use a config array like this. Please take a look at how similar adapters are implemented in this repo to see how the configuration should be implemented.